apps/studio/src/lib/db/clients/utils.ts

```diff
@@ -233,6 +233,11 @@ export async function withClosable<T>(item, func): Promise<T> {
 
 }
 
+export function checkValidityOfDatabaseName(databaseName: string): boolean {
+  const dbNameRegex = /^[a-zA-Z0-9_-]*$/
```

Comment:
is this always true?

Also - don't need this in the front end as you do it here

Reply:
Was taking the approach of validate on front end and "back end" This is going to be going away anyhow. Like it never happened